### PR TITLE
Go 1.20 and some updates

### PR DESCRIPTION
**Pull request by roman-khimov**

### CODE REVIEW

**Comment by cthulhu-rider** [comment]
all LGTM. Maybe insta release?

**Comment by roman-khimov** [comment]
That was intended to be the next PR.

**Review by cthulhu-rider — APPROVED** [review]

File: .github/workflows/golangci-lint.yml [hunk 1]
```diff
@@ -13,13 +13,13 @@ jobs:
     name: lint
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-go@v4
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
         with:
-          go-version: '1.19'
+          go-version: '1.20'
```
Comment (carpawell) [inline .github/workflows/golangci-lint.yml:19]: https://github.com/nspcc-dev/neofs-node/commit/4de9938c0375a04c8577b2168f32d8f9c764f25f ?

**Review by carpawell — APPROVED** [review]

File: .github/workflows/golangci-lint.yml (hunk 1 above)
Comment (roman-khimov) [reply .github/workflows/golangci-lint.yml]: Good one, added that.

**Review by cthulhu-rider — APPROVED** [review]

**Review by carpawell — APPROVED** [review]

**Pull request merged**